src/redirect.rs

```diff
@@ -0,0 +1,74 @@
+use std::str::Utf8Error;
+
+use rocket::{http::RawStr, request::FromParam};
+
+pub struct Destination {
+    pub uri: &'static str,
+}
+
+impl<'r> FromParam<'r> for Destination {
+    type Error = NoRedirectFound;
+
+    fn from_param(param: &'r RawStr) -> Result<Self, Self::Error> {
+        let uri = match param.percent_decode()?.as_ref() {
+            "community.html" => "/community",
+            "conduct.html" => "/policies/code-of-conduct",
+            "contribute-bugs.html" => "/community",
+            "contribute-community.html" => "/governance/teams/community",
+            "contribute-compiler.html" => "/governance/teams/language-and-compiler",
+            "contribute-docs.html" => "/governance/teams/documentation",
+            "contribute-libs.html" => "/governance/teams/library",
+            "contribute-tools.html" => "/governance/teams/dev-tools",
+            "contribute.html" => "/community",
+            "documentation.html" => "/learn",
+            "downloads.html" => "/tools/install",
+            "friends.html" => "/production",
+            "index.html" => "/",
+            "install.html" => "/tools/install",
+            "legal.html" => "/policies",
+            "other-installers.html" => {
+                "https://forge.rust-lang.org/other-installation-methods.html"
+            }
+            "security.html" => "/policies/security",
+            "team.html" => "/governance",
```

Comment:
currently both 
https://www.rust-lang.org/en-US/team.html
https://www.rust-lang.org/en-US/governance
does not work
What should be used in the book?

Reply:
The current URL is https://www.rust-lang.org/governance